src/app/@ansyn/ansyn/modules/plugins/app/context-menu/context-menu.plugin.ts

```diff
@@ -64,7 +64,7 @@ export class ContextMenuPlugin extends BaseImageryPlugin {
 			tap(([point, overlaysState]) => {
 				const overlays = overlaysState.filteredOverlays
 					.map((id: string): IOverlay => overlaysState.entities[id])
-					.filter(({ footprint }) => inside(point, footprint));
+					.filter(({ footprint }) => booleanContains(footprint, point));
```

Comment:
use @turf/boolean-point-in-polygon
https://github.com/Turfjs/turf-inside

Reply:
footprint is not longer only `MultiPolygon` it could be `Point` or `LineString` and `booleanPointInPolygon` work only with `Point` and `Polygon`